<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GATEWAY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIp">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.31</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>00000; </string>
    </property>
    <property name="text">
     <string>1470</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxConnect">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
